### PR TITLE
Add OS & optional EULA licenses

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
@randomir to unblock you for tomorrow. The main page (screenshot below) sits under ....../en/latest/licenses.html

image

image

### CODE REVIEW

File: docs/licenses/dnx.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+==============
+dwave-networkx
+==============
+
+.. include:: ../docs_dnx/LICENSE.txt
```
Comment (randomir) [inline docs/licenses/dnx.rst]: :smile:

Comment (JoelPasvolsky) [reply docs/licenses/dnx.rst]: Yes, it is the odd man out

**Comment by JoelPasvolsky** [comment]
@randomir, added a EULA for you. The link to this EULA is https://docs.ocean.dwavesys.com/en/latest/licenses/drivers and any future EULAs will have a similar url, just different last part.

**Comment by JoelPasvolsky** [comment]
@randomir, please check update for inspector and then this can be merged

**Pull request merged**